contrib/sip_proxy/filters/network/source/decoder.h

```diff
@@ -116,8 +93,7 @@ class DecoderCallbacks {
    */
   virtual DecoderEventHandler& newDecoderEventHandler(MessageMetadataSharedPtr metadata) PURE;
   virtual absl::string_view getLocalIp() PURE;
-  virtual std::string getOwnDomain() PURE;
-  virtual std::string getDomainMatchParamName() PURE;
+  virtual std::shared_ptr<SipSettings> settings() PURE;
```

Comment:
Would it make sense to mark methods that don't mutate the object's state as `const`?

Reply:
done, add `const`.